custom-crawls/lessons/lesson-name.md

```diff
@@ -0,0 +1,11 @@
+# Lesson: Name
```

Comment:
You can delete this if you aren't using separate lessons for now!

Reply:
Sounds good! Will do.